LayoutTests/TestExpectations

```diff
@@ -7044,6 +7044,9 @@ imported/w3c/web-platform-tests/css/css-view-transitions/scroller-child.html [ I
 imported/w3c/web-platform-tests/css/css-view-transitions/scroller.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-view-transitions/iframe-and-main-frame-transition-old-main-new-iframe.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-view-transitions/iframe-and-main-frame-transition-old-main-old-iframe.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe.html [ ImageOnlyFailure ]
```

Comment:
No bug for this one?

Reply:
I try to not file bugs until we know what the concrete issue is